ui/package.json

```diff
@@ -113,16 +112,19 @@
     "fs-extra": "^10.0.0",
     "html-webpack-plugin": "4.5.2",
     "husky": "^7.0.4",
+    "immer": ">=9.0.6",
```

Comment:
I'm not sure this helps. AFAIK, `npm` will still use embedded `immer` versions declared in transitive dependencies for their respective "client" modules.